bench/bench-tbl_summary.R

```diff
@@ -17,3 +18,9 @@ bench::mark(
   check = FALSE,
   min_iterations = 30
 )
+
+# knitr::knit(input = "bench/benchmark2html.Rmd", output = "bench/benchmark2html.html", envir = new.env())
+
+rmarkdown::render(input = "benchmark2html.Rmd", # IMPORTANT: Relative paths doesn't work maybe solvable with {here}
```

Comment:
yes, I think `here::here()` will be perfect here.

Reply:
OK, added a call to here::here(), seems to work perfectly.